packages/inula-adapter/src/vue/globalAPI.tsx

```
@@ -0,0 +1,336 @@
+      } else if (plugin && typeof plugin.install === 'function') {
+        installedPlugins.add(plugin);
+        plugin.install(app, ...options);
+      } else if (typeof plugin === 'function') {
+        installedPlugins.add(plugin);
+        plugin(app, ...options);
+      }
+
+      return app;
+    },
+
+    mixin(mixin) {
```

ESLint

[error] 'mixin' is defined but never used.

(@typescript-eslint/no-unused-vars)